Alignment/APEEstimation/python/SectorBuilder_Tec_cff.py

```diff
@@ -129,47 +129,47 @@
 )
 TecMinusRing1Stereo = TecMinusRing1.clone(
     name = 'TecMinusRing1Stereo',
-    isRPhi = [2],
+    isStereo = [1],
```

Comment:
This was actually not a bug, but is a clarification of the code. Previously stereo modules were assigned by taking those that are not RPhi (see https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/DataFormats/SiStripDetId/interface/TIBDetId.h#L103-L108).  We now introduced a isStereo entry in the TrackerTreeGenerator output to have more clearly documented which modules are which. At least for the sectors where we split between stereo and rPhi the results are the same.